.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
We only create tags against code landed in master by convention. So this action runs whenever a tag against master is created.